src/doc/dev_manual/SiteReliabilityEngineering.rst

```diff
@@ -193,7 +193,7 @@ SRE vs. Product Development
 
 Part of the reason for formalizing this process is the recognition of a
 different category of work,
-`Site Reliability Engineering <https://en.wikipedia.org/wiki/Site_Reliability_Engineering>`__ (SRE),
+`Software Reliability Engineering <https://en.wikipedia.org/wiki/Site_Reliability_Engineering>`__ (SRE),
```

Comment:
What is the purpose of this change? The wikipedia page is for `Site Reliability Engineering` and the RST file we are in is `Site Reliability Engineering` as well.

Reply:
The purpose is to change `Site` to `Software` in our docs. Wherever SRE is spelled out in words, I think its best to have it read `Software Reliability Engineering`. It reads this way all other places in that `.rst` file except when we are referring to Google's original title for this kind of work.

I think `Site` is confusing for people.

Reply:
Makes sense. 👍